src/commonMain/kotlin/fr/acinq/lightning/channel/ShuttingDown.kt

```diff
@@ -174,12 +174,15 @@ data class ShuttingDown(
                 else -> unhandled(cmd)
             }
             is ChannelCommand.WatchReceived -> when (val watch = cmd.watch) {
-                is WatchEventSpent -> when (watch.tx.txid) {
-                    commitments.remoteCommit.txid -> handleRemoteSpentCurrent(watch.tx)
-                    commitments.remoteNextCommitInfo.left?.nextRemoteCommit?.txid -> handleRemoteSpentNext(watch.tx)
-                    else -> handleRemoteSpentOther(watch.tx)
+                is WatchEventConfirmed -> when (val res = acceptFundingTxConfirmed(watch)) {
```

Comment:
Because the `unhandled` function has mostly been used to detect things we forgot to handle...it feels hacky to do this in an `unhandled`, but I can probably factor a default handler for `WatchConfirmed`, I'll try something.

Reply:
Factored in https://github.com/ACINQ/lightning-kmp/pull/422/commits/90cea50781bbd807f6e8be563141b09fc8c56d7d
There is an ugly cast to `PersistedChannelState`, but that will disappear with #426